adam/perception/perception_graph.py

```diff
@@ -73,6 +73,31 @@
 from adam.perception.high_level_semantics_situation_to_developmental_primitive_perception import (
     HighLevelSemanticsSituationToDevelopmentalPrimitivePerceptionGenerator,
 )
+from adam.perception.perception_graph_nodes import (
+    PerceptionGraphNode,
+    UnwrappedPerceptionGraphNode,
+    ObjectNode,
+    CategoricalNode,
+    ContinuousNode,
+    RgbColorNode,
+)
+from adam.perception.perception_graph_predicates import (
+    NodePredicate,
+    GeonPredicate,
+    CrossSectionPredicate,
+    RegionPredicate,
+    AxisPredicate,
+    AnyObjectPerception,
+    IsOntologyNodePredicate,
+    IsColorNodePredicate,
+    ObjectSemanticNodePerceptionPredicate,
+    IsPathPredicate,
+    PathOperatorPredicate,
+    AnyObjectPredicate,
+    CategoricalPredicate,
+    ContinuousPredicate,
+    RgbColorPredicate,
+)
```

Comment:
Agreed, for some reason I recall the auto sorting of imports on this repository not working. If I get some free time to address this I will but otherwise I feel inclined to not worry about this for the moment.